src/System.Private.Jit/src/Internal/Runtime/JitSupport/JitCompilation.cs

```diff
@@ -52,11 +52,7 @@ public ObjectNode GetFieldRvaData(FieldDesc field)
 
         internal MethodIL GetMethodIL(MethodDesc method)
         {
-            // Flush the cache when it grows too big
```

Comment:
This will not go through the cache anymore ... ok for JIT prototype, but it will likely need to be fixed eventually.

Reply:
Yeah, I didn't want to copy paste the cache. I don't like the current sharing plan between `JitCompilation` and `Compilation` (because the sharing plan is copy-paste). I think we should be able to at least do dot-file sharing, but I haven't thought about it too much yet. This file will have to be reconciled with `Compilation`.